tests/jax/models/GPT2/test_gpt2.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Sequence, Dict
+
+import jax
+import pytest
+from flax import linen as nn
+from infra import ModelTester, RunMode
+from transformers import AutoTokenizer, FlaxGPT2LMHeadModel
+
+MODEL_PATH = "openai-community/gpt2"
+
+
+class GPT2Tester(ModelTester):
+    """Tester for GPT2 for autoregressive text generation."""
+
+    @staticmethod
```

Comment:
Forgot to remove here. Please make sure that you've removed `@staticmethod` in all places where overriding methods which shouldn't be static, and add `# @override`